README.md

```diff
@@ -28,9 +28,78 @@ Examples:
 * [Basic example](https://jywarren.github.io/image-sequencer/)
 * [NDVI example](https://jywarren.github.io/image-sequencer/examples/ndvi/) - related to [Infragram.org](http://infragram.org)
 
+Using the library:
+The Image Sequencer Library exports a function ImageSequencer which initializes a sequencer.
+```js
```

Comment:
Sorry to be picky here but I think clear empty lines between paragraphs is a good thing for readability -- especially when on a command line interface and reading the raw Markdown. Thanks!